src/components/sbb-tag-group/sbb-tag-group.e2e.ts

```diff
@@ -1,588 +1,558 @@
           <sbb-tag-group value="tag2">
             <sbb-tag id="sbb-tag-1" value="tag1">Tag 1</sbb-tag>
             <sbb-tag id="sbb-tag-2" value="tag2">Tag 2</sbb-tag>
             <sbb-tag id="sbb-tag-3" value="tag3">Tag 3</sbb-tag>
           </sbb-tag-group>
         `);
-        element = await page.find('sbb-tag-group');
-
-        await page.waitForChanges();
 
-        expect(await element.getProperty('value')).toEqual('tag2');
-        expect(await element.find('sbb-tag#sbb-tag-1')).not.toHaveAttribute('checked');
-        expect(await element.find('sbb-tag#sbb-tag-2')).toHaveAttribute('checked');
-        expect(await element.find('sbb-tag#sbb-tag-3')).not.toHaveAttribute('checked');
+        expect(element.value).to.be.equal('tag2');
+        expect(element.querySelector('sbb-tag#sbb-tag-1')).not.to.have.attribute('checked');
+        expect(element.querySelector('sbb-tag#sbb-tag-2')).to.have.attribute('checked');
+        expect(element.querySelector('sbb-tag#sbb-tag-3')).not.to.have.attribute('checked');
       });
 
       it('should read value [prop]', async () => {
-        page = await newE2EPage();
-        await page.setContent(`
+        element = await fixture(html`
           <sbb-tag-group>
             <sbb-tag id="sbb-tag-1" value="tag1">Tag 1</sbb-tag>
             <sbb-tag id="sbb-tag-2" value="tag2">Tag 2</sbb-tag>
             <sbb-tag id="sbb-tag-3" value="tag3">Tag 3</sbb-tag>
-          </sbb-tag-group>        `);
-        element = await page.find('sbb-tag-group');
-        element.setProperty('value', 'tag2');
+          </sbb-tag-group>
+        `);
 
-        await page.waitForChanges();
+        element.value = 'tag2';
+        await element.updateComplete;
 
-        expect(await element.getProperty('value')).toEqual('tag2');
-        expect(await element.find('sbb-tag#sbb-tag-1')).not.toHaveAttribute('checked');
-        expect(await element.find('sbb-tag#sbb-tag-2')).toHaveAttribute('checked');
-        expect(await element.find('sbb-tag#sbb-tag-3')).not.toHaveAttribute('checked');
+        expect(element.value).to.be.equal('tag2');
+        expect(element.querySelector('sbb-tag#sbb-tag-1')).not.to.have.attribute('checked');
+        expect(element.querySelector('sbb-tag#sbb-tag-2')).to.have.attribute('checked');
+        expect(element.querySelector('sbb-tag#sbb-tag-3')).not.to.have.attribute('checked');
       });
 
       it('should ignore because value is an array', async () => {
-        page = await newE2EPage();
-        await page.setContent(`
-          <sbb-tag-group></sbb-tag-group>
-        `);
-        element = await page.find('sbb-tag-group');
-        element.setProperty('value', []);
-
-        await page.waitForChanges();
+        element = await fixture(html` <sbb-tag-group></sbb-tag-group> `);
+        element.value = [];
 
-        expect(await element.getProperty('value')).toEqual([]);
+        await element.updateComplete;
+        expect(element.value).to.be.empty;
```

Comment:
I think empty and empty array is not the same?

Reply:
It should work, but I can add an additional check (`.to.be.an('array').that.is.empty;`) as suggested by the [doc](https://www.chaijs.com/api/bdd/#method_empty:~:text=a%20target%E2%80%99s%20type.-,expect(%5B%5D).to.be.an(%27array%27).that.is.empty%3B,-Add%20.not).